workspaces/mta/.changeset/two-fishes-cough.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/backstage-plugin-catalog-backend-module-mta-entity-provider': minor
```

Comment:
```suggestion
'@backstage-community/backstage-plugin-catalog-backend-module-mta-entity-provider': patch
```

Reply:
The code that's changing is just removing logging, not sure that a minor changes makes sense. Same with the one below.